bin/feature

```diff
@@ -199,6 +199,9 @@ when 'merge'
    exit 1 unless confirm("Merge feature branch named: '#{feature}' ?")
 
    update = Git::submodules_update("get")
+   pull_desc = pull_info[:description].shellescape
+   failure_message = highlight("Merge conflicts detected, merge #{dev_branch} into #{feature} and resolve conflicts.".shellescape)
```

Comment:
Maybe `merge_failure_message`? There are _lots_ of git commands run in a row but this is the error message just for the merge command in particular.